src/ext/java/org/opentripplanner/ext/transmodelapi/mapping/PreferencesMapper.java

```diff
@@ -24,6 +31,20 @@ static void mapPreferences(
         setStreetReluctance(preferences, streetReluctance);
       }
     );
+
+    if (GqlUtil.hasArgument(environment, "maxAccessEgressDurationForMode")) {
+      preferences.withStreet(street -> {
+        for (var entry : (List<Map<String, ?>>) environment.getArgument(
+          "maxAccessEgressDurationForMode"
+        )) {
```

Comment:
It is usually cleaner to be explicit with the type, when there is a need to cast
```suggestion
        for (List<Map<String, ?>> entry : environment.getArgument(
          "maxAccessEgressDurationForMode"
        )) {
```